vos/vos/commands/vtag.py

```diff
@@ -107,14 +108,16 @@ def vtag():
         nargs="*")
     parser.add_option('--remove', action="store_true",
                       help='remove the listed property')
+    parser.add_option('-R', '--recursive', action="store_true",
```

Comment:
The `vrm` flag was a lower-case `r`. Should the flags be consistent?

Reply:
Changed `vrm`'s flat to upper case. `rm` supports both but they are equivalent. `v*` commands will only support `-R`.